packages/react-native-executorch/common/rnexecutorch/models/text_to_image/Constants.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <string>
+
+namespace rnexecutorch::models::text_to_image::constants {
+
+inline constexpr std::string kBosToken = "<|startoftext|>";
```

Comment:
I don't think this will work, you should probably make this `std::string_view` to use it in constexpr context